spring-boot-modules/spring-boot-3/src/test/java/com/baeldung/springjunitconfiguration/Person.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.springjunitconfiguration;
+
+public class Person {
```

Comment:
Can you move this class to the src/main folder instead of src/test folder?

Reply:
Hi @Maiklins, I've moved it.